doc/samtools-faidx.1

```diff
@@ -124,6 +127,12 @@ Read/Write to specified compressed file index (used with .gz files).
 .TP
 .B -h, --help
 Print help message and exit.
+.TP
+.BI --output-fmt-option\  OPT=VAL
+Set the output format options, level=0..9 for compression level 0 to 9.
```

Comment:
Change this to 1 to 9, as level 0 is parsed as default compression (level 6 atm).  We could change that to be uncompressed, but it's pointless as you could just not ask for .gz in the filename then.

Reply:
there was the error of skipping 0 which is now fixed.
retaining 0..9 as it is the allowed range and as the same is supported by other samtools commands.
with 0, it will be uncompressed data in bgzf format. 
default continues to be set by underlying compression library.